tensorflow/python/training/slot_creator.py

```diff
@@ -80,12 +81,13 @@ def create_slot(primary, val, name, colocate_with_primary=True):
     A `Variable` object.
   """
   # Scope the slot name in the namespace of the primary variable.
-  with ops.name_scope(primary.op.name + "/" + name) as scope:
-    if colocate_with_primary:
-      with ops.colocate_with(primary):
-        return _create_slot_var(primary, val, scope)
-    else:
-      return _create_slot_var(primary, val, scope)
+  #with variable_scope.variable_scope(primary.op.name + "/" + name) as scope:
```

Comment:
Maybe remove this comment and leave the name_scope as it was, i.e., "with ops.name_scope(scope_name):".